alluxio/option.py

```diff
@@ -145,9 +145,25 @@ def json(self):
 class GetStatus(_JsonEncodable):  # pylint: disable=too-few-public-methods
     """Options to be used in :meth:`alluxio.Client.get_status`.
 
-    Currently, it is an empty class, options may be added in future releases.
+    Args:
+        load_metadata_type (:class:`alluxio.wire.LoadMetadataType`): The type of
+            loading metadata, can be one of
+            :data:`alluxio.wire.LOAD_METADATA_TYPE_NEVER`,
+            :data:`alluxio.wire.LOAD_METADATA_TYPE_ONCE`,
+            :data:`alluxio.wire.LOAD_METADATA_TYPE_ALWAYS`,
+            see documentation on :class:`alluxio.wire.LoadMetadataType` for more
+            details
     """
 
+    def __init__(self, **kwargs):
+        self.load_metadata_type = kwargs.get('load_metadata_type')
```

Comment:
Good question. Personally I prefer being explicit here, instead of letting the server to decide, if I send a request without params.

However, I would probably not include such a change into this same PR, otherwise its' going to be not coherent with existing classes:
- GetStatus
- ListStatus
- .. more from `option.py`

At the moment, none of those seem to have explicit defaults, and I would either keep it all as-is or add defaults everywhere, at once.

Makes sense?